src/commonMain/kotlin/com.adamratzman.spotify/SpotifyExceptions.kt

```diff
@@ -0,0 +1,4 @@
+/* Spotify Web API - Kotlin Wrapper; MIT License, 2019; Original author: Adam Ratzman */
+package com.adamratzman.spotify
+
+open class SpotifyException(message: String, cause: Throwable? = null) : Exception(message, cause)
```

Comment:
I really like marking such classes as `sealed` as it simplifies the handling of those.